strapi/strapi
# chore(content-releases): Create releases CT, create and find services and controllers

### What does it do?

This PR create the new releases and release action content types, also it creates two new services/controllers for create a release and to find all the releases.

⚠️ This PR doesn't handle the ability to add an entry to a specific release yet, for now its scope is only create a release with its name and the definition of the content types.

### How to test it?

You need to have a right EE license with the feature enabled and also the `FEATURE_FLAG_CONTENT_RELEASES` env variable set to true.

1. Try to create a new release with the endpoint: POST `/content-releases`
- ✅  If you send on the body an object with the field `name` and a name everything should work without problems
- 🔴  If you are not a super admin user, you should see an error
- 🔴  If you send an empty error, you should see an error

2. Try to get the list of all the releases with the endpoint: GET `/content-releases`
